src/sidebar/services/test/annotations-exporter-test.js

```diff
@@ -36,4 +42,84 @@ describe('AnnotationsExporter', () => {
       annotations: [firstBaseAnnotation, secondBaseAnnotation],
     });
   });
+
+  describe('buildTextExportContent', () => {
+    it('throws error when empty list of annotations is provided', () => {
+      assert.throws(
+        () => exporter.buildTextExportContent([]),
+        'No annotations to export',
+      );
+    });
+
+    it('generates text content with provided annotations', () => {
+      const isoDate = now.toISOString();
+      const annotation = {
+        ...newAnnotation(),
+        ...publicAnnotation(),
+        created: isoDate,
+      };
+      // Title should actually be an array
+      annotation.document.title = [annotation.document.title];
```

Comment:
I think we have discussed this a couple of time, that the API defines `document.title` as an array when returned, but as a string when saved.

I worked around it here, but we probably want to address that eventually.